R/main.R

```diff
@@ -1,10 +1,20 @@
-#' shiny.i18n: A package for Shiny applications internationalization.
+#' i18n
 #'
-#' The shiny.i18n package provides functions for easy Shiny applications internationalization.#'
+#' Returns a translator object which gives you translation
+#' method of your functions.
 #'
-#' @section Foo functions:
-#' The foo functions ...
+#' @param translation_file character path to transtaltion file
+#' @param key_translation character with language code, e.g. "en", "pl".
+#' If NULL, LANG environment variable will be loaded.
```

Comment:
this is not precise. The code uses two first characters of the LANG env variable, right?